docs/content/preview/yugabyte-platform/security/ldap-authentication-platform.md

```diff
@@ -136,3 +136,24 @@ Consider the following example:
     ```output
     You are connected to database "exampledb" as user "adam" on host "localhost" at port "5433".
     ```
+
+## Troubleshooting
```

Comment:
This subsection needs to be extracted and defined as a new page under Troubleshooting: https://docs.yugabyte.com/preview/yugabyte-platform/troubleshoot/

Then we can include a link to it from the current document.